llvm/test/Instrumentation/InstrProfiling/binary-correlate.ll

```diff
@@ -0,0 +1,11 @@
+; RUN: opt < %s -passes=instrprof -profile-correlate=binary -S | FileCheck %s
```

Comment:
This isn't clear how `__profd_foo` is different from `-profile-correlate=none`. This test can be added to `coverage.ll` as a new RUN line